templates/data/spark/index.html

```diff
@@ -231,6 +320,28 @@ <h2>Consulting services</h2>
     </div>
   </section>
 
+  <section class="p-section">
+    <div class="grid-row--50-50">
+      <hr class="p-rule" />
+      <div class="grid-col">
+        <h2>Customer story</h2>
+      </div>
+      <div class="grid-col">
+        <div class="u-embedded-media">
+          <iframe width="560" height="315" src="https://www.youtube.com/embed/JKZKpXaJsPI?si=bWMlnywbkw6N0Z1C"
+            title="YouTube video player" frameborder="0"
+            allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
+            referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
+        </div>
```

Comment:
Can you please update this to use the lite-youtube macro

https://github.com/canonical/canonical.com/blob/main/templates/macros/_macro-lite-video.jinja

This ensures the YouTube embed scripts are not loaded until it is clicked which helps in increasing page load speed and performance.

<img width="1970" height="910" alt="Image" src="https://github.com/user-attachments/assets/384240bc-8818-43c9-a979-af97a35ae986" />